api/src/paths/dwc/spatial/metadata.ts

```diff
@@ -50,9 +55,9 @@ GET.apiDoc = {
  *
  * @returns {RequestHandler}
  */
-export function getSpatialMetadataById(): RequestHandler {
+export function getSpatialMetadataBySubmissionSpatialComponentIds(): RequestHandler {
   return async (req, res) => {
-    const submissionSpatialComponentId = Number(req.params.submissionSpatialComponentId);
+    const submissionSpatialComponentIds = ((req.query.submissionSpatialComponentIds || []) as string[]).map(Number);
```

Comment:
style consideration:

I think  const submissionSpatialComponentIds = ((req.query.submissionSpatialComponentIds || []) as string[]).map((item) =>
      Number(item)
    );
    is more readable.

Reply:
No preference really, either is fine. Your way Anissa is maybe a bit easier to read, for someone not familiar with the shorthand, but I think its fine its fine either way.

Reply:
Per discussion with Nick, `Number` constructor is acceptable to use in lieu of arrow function. See https://stackoverflow.com/questions/48343478/what-does-mapnumber-do-here